src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
Actually, now that I think about it. I'm not sure I agree that `next` and `previous` are `player` actions. `next` and `previous` are exclusively playlist actions. They have no meaning unless a playlist is associated. In the API example above, there's `playlistItem` which has `playlist` in it's name and all the comments about what the methods do refer to actions on the playlist.
It does not make sense to `next` a player unless the player has playlist functionality built-in.